classes/01class/exercises/c01-git03/raghunadhpokkalath/README.md

```diff
@@ -0,0 +1,26 @@
+# c01-git03
+
+1. Did you notice any difference in the size of the repo before and after adding the big file?
+```
+Yes after adding the 10 MB file the size of directory became  20 MB
+```
+
+2. What is the reason for this problem?
+```
+ When a new file is added a copy is maintained by git inside .git folder
+```
+
+3. How do you remove something from Git history after it is pushed to the remote repository? Which commands would you use? 
+
+```
+Using git filter-branch command
```

Comment:
Have a look at `filter-repo`, it is recommended instead of `filter-branch`, simpler and safer.
 https://github.com/newren/git-filter-repo

Reply:
Hi @denstorti  modified to use filter-repo